service/solace-plugin/pom.xml

```diff
@@ -40,6 +40,11 @@
             <artifactId>solace-messaging-client</artifactId>
             <version>${solace-messaging-client.version}</version>
         </dependency>
+        <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
+            <version>2.6</version>
+        </dependency>
```

Comment:
Same comment 
This is the newer version and I think we MUST switch to this : https://mvnrepository.com/artifact/org.apache.commons/commons-lang3/3.16.0